src/screens/profile-screen/Metrics.tsx

```diff
@@ -0,0 +1,55 @@
+import { ProfileUser } from 'audius-client/src/pages/profile-page/store/types'
+import { Text, View } from 'react-native'
+
+import { makeStyles } from 'app/styles/makeStyles'
+
+const useStyles = makeStyles(({ typography, palette, spacing }) => ({
+  root: {
+    display: 'flex',
```

Comment:
No need for `display: 'flex'` in react-native, it's the default display

Reply:
ohh, great!